### PR TITLE
Removed Hardcoded Admin Username & Password

### PR DESCRIPTION
**Pull request by Jay24004**
this PR removed the hardcoded admin user and password and add use of environment variables for admin username and password

### CODE REVIEW

File: src/pages/admin/index.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
-"use client";
-import { useState, useEffect } from "react";
+import { useState, useEffect, use } from "react";
```
Comment (ratishjain12) [inline src/pages/admin/index.tsx]: Remove import use from this file as I dont see the use of it :)

Can you make it in client side only but just converting it to NEXT_PUBLIC convention 

Also no need to use dotenv

@Jay24004

**Comment by Jay24004** [comment]
@ratishjain12, take a look here.

**Comment by ratishjain12** [comment]
I have requested some changes finish those and I'll merge it :)

**Comment by Jay24004** [comment]
> I have requested some changes finish those and I'll merge it :)

where do you request those?

**Comment by ratishjain12** [comment]
> > I have requested some changes finish those and I'll merge it :)
> 
> where do you request those?

you can check those in pending review changes or go changed files tab

**Comment by Jay24004** [comment]
> > > I have requested some changes finish those and I'll merge it :)
> > 
> > 
> > where do you request those?
> 
> you can check those in pending review changes or go changed files tab

i don't think there are any changes requested can you check ?

**Comment by ratishjain12** [comment]
image

**Review by ratishjain12 — CHANGES_REQUESTED** [review]
Remove import use from this file as I dont see the use of it :)

Can you make it in client side only but just converting it to NEXT_PUBLIC convention

Also no need to use dotenv

@Jay24004

File: src/pages/admin/index.tsx (hunk 1 above)
Comment (Jay24004) [reply src/pages/admin/index.tsx]: I have removed the import `use` and dotenv
also, the Admin username/password should be server-side only as they are sensitive information

**Review by ratishjain12 — APPROVED** [review]

**Pull request merged**